harness/config/pipeline.yml

```
@@ -4,6 +4,33 @@ command('app publish'): |
   passthru docker-compose push @('pipeline.publish.services')
   run docker logout @('docker.registry.url')
 
+command('app publish chart <release> <message>'):
+  env:
+    SSH_PRIVATE_KEY:    = @('pipeline.publish.chart.git.ssh_private_key')
+    REPOSITORY: = @('pipeline.publish.chart.git.repository')
+    ARTIFACTS_PATH:  = "./build-artifacts-repository/" ~ @('pipeline.publish.chart.git.path') ~ "/" ~ input.argument('release')
+    MESSAGE:    = input.argument('message')
+    USER_EMAIL: = @('pipeline.publish.chart.git.email')
+  exec: |
+    #!bash(workspace:/)|@
+
+    run rm -rf build-artifacts-repository
+    echo "${SSH_PRIVATE_KEY}" | base64 -d > id_rsa
+    chmod 0600 id_rsa
+
+    export GIT_SSH_COMMAND='ssh -i ./id_rsa -o "IdentitiesOnly yes" -F /dev/null -o StrictHostKeyChecking=no'
+
+    run git clone "$REPOSITORY" ./build-artifacts-repository
+    run git -C ./build-artifacts-repository config user.email "${USER_EMAIL}"
+
+    run mkdir -p $ARTIFACTS_PATH
+    run rsync --exclude='*.twig' --exclude='_twig' --delete -a .my127ws/helm/app/ "${ARTIFACTS_PATH}/"
+
+    export GIT_SSH_COMMAND='ssh -i ../id_rsa -o "IdentitiesOnly yes" -F /dev/null -o StrictHostKeyChecking=no'
+    run git -C ./build-artifacts-repository add .
+    run "git -C ./build-artifacts-repository commit --allow-empty -m '${MESSAGE}'"
+    run git -C ./build-artifacts-repository push origin master
```

To import a 0.11.0 php harness change:
```suggestion
    run git -C ./build-artifacts-repository push origin -u HEAD
```